exporter/sentryexporter/sentry_exporter_test.go

```diff
@@ -170,6 +169,72 @@ func generateOrphanSpansFromSpans(spans ...*sentry.Span) []*sentry.Span {
 	return orphanSpans
 }
 
+func TestSpanEventToSentryEvent(t *testing.T) {
+	t.Run("Exception Event with both exception type and message", func(t *testing.T) {
```

Comment:
Can we add table tests where the exception type or exception message is empty? Just to say that these are valid arguments to pass into the func?